### PR TITLE
Wip/hughsie/win32 console

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
Untitled

**Review by superm1 — DISMISSED** [review]

File: src/fu-tool.c [hunk 1]
```diff
@@ -3545,11 +3546,12 @@ main(int argc, char *argv[])
 	fu_util_cmd_array_sort(cmd_array);
 
 	/* non-TTY consoles cannot answer questions */
-	priv->interactive = isatty(fileno(stdout)) != 0;
```
Comment (superm1) [inline src/fu-tool.c:3548]: This also goes on fu-util right?

Comment (hughsie) [reply src/fu-tool.c:3548]: Not sure I understand the question, sorry.

Comment (superm1) [reply src/fu-tool.c:3548]: Don't we also have an interactive check there too?

Comment (hughsie) [reply src/fu-tool.c:3548]: Ahh! Indeed we do. Fixed.

**Pull request merged**